friends/src/components/FriendForm.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import { UserForm, UserFormInput, UserFormBtn } from './styles/S_FriendForm'
+
```

Comment:
Best practice for handling form is to keep the input state on the form component instead of the parent component. Just make the form component to receive submit handler as props to handle onSubmit event.